example/README.md

```diff
@@ -85,22 +85,32 @@ Do *not* run `direnv allow` for now, or if you already did so run
 
 ## Set up `lorri`
 
-`lorri` can watch a `shell.nix` file, together with all its
-dependencies. If any of these files change, `lorri` starts nix and
-builds the new version.
+`lorri daemon` starts a process that watches one or more `shell.nix` files,
+together with all its dependencies. If any of these files change, the
+daemon starts nix and builds the new version.
 
 Install `lorri` by following the instructions [in the
 README](../README.md#install).
 
-Now that you have installed `lorri`, let’s take it for a spin.
-Start up the `lorri` watcher:
+Now that you have installed `lorri`, let’s take it for a spin. Start up the
+`lorri` daemon (note that it does not matter which directory this command is
+run in):
 
 ```bash
-$ lorri watch
+$ lorri daemon
+```
+
+As you can see, the daemon does nothing initially. It is waiting to be told
+about projects. There is a "plumbing" command `lorri ping_` to do this manually
```

Comment:
```suggestion
about projects. The "plumbing" command `lorri ping_` tells the daemon to start watching a project’s nix file. It can be used in scripts, or, like we do here, to assure yourself that the daemon is working correctly.
```